ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml

```diff
@@ -51,6 +51,18 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: libvirt-installer
+  openstack-cinder-csi-driver-operator:
```

Comment:
@jsafrane do I need this one you think?

Reply:
No, it's not used by any CI step. The operator image is used by installer and it's baked into OCP release already. I think.